addons/mrp/tests/test_order.py

```diff
@@ -615,7 +615,7 @@ def test_product_produce_4(self):
         self.assertEqual(len(ml_p2), 1)
 
         # Add some quantity already done to force an extra move line to be created
-        ml_p1[0].qty_done = 1.0
+        # ml_p1[0].qty_done = 1.0
```

Comment:
@amoyaux This is the only test that still failed with unit_factor computed. But I think the test should not behave like it does. Could you take a look at it ? Thanks